app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -22,6 +22,10 @@ class ManageIQ::Providers::Redhat::InfraManager < ManageIQ::Providers::InfraMana
 
   supports :provisioning
   supports :refresh_new_target
+  supports :vm_import do
```

Comment:
As far  as  I know the proper way to use the :supports method is with unsupported_reason_add, please see an example here:
https://github.com/ManageIQ/manageiq-providers-ovirt/blob/master/app/models/manageiq/providers/redhat/infra_manager/vm/operations/snapshot.rb#L5

Reply:
Done.